### PR TITLE
Awesome book with ES6

### PR DESCRIPTION
**Pull request by Shaheryar0054**
- Used ES6 syntax
- Used Arrow function
- Create a new directory and initialize it with git.
- Use the npm init -y command to create a package.json file.
- Create the entry point for your JavaScript code called index.js
- Create an index.css file for your styles.
- Create the main index.html file. Link your index.js and index.css files there.
- Create a directory called modules.

### CODE REVIEW

File: modules/store.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    const books = store.getBooks();
+    books.forEach((book, index) => {
+      if (book.authore === authore) {
+        books.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```
Comment (HAMMAS-SALEEM) [inline modules/store.js]: image

- [ ] You methods in the class are not arrow methods :fearful: Kindly convert them into arrow methods as i have done in the image above :arrow_up: to follow the ES6 :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Shaheryar0054 :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github Flow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/store.js (hunk 1 above)
Comment (Whoistolu) [reply modules/store.js]: - Well resolved @Shaheryar0054 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @Shaheryar0054 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**